packages/blade/src/components/Tabs/TabList.web.tsx

```diff
@@ -30,7 +35,7 @@ const TabList = ({ children }: { children: React.ReactNode }): React.ReactElemen
   }, [children, selectedValue, setSelectedValue]);
 
   return (
-    <BaseBox display={isVertical ? 'flex' : 'block'} flexShrink={0} overflow="hidden">
+    <Box {...props} display={isVertical ? 'flex' : 'block'} flexShrink={0} overflow="hidden">
```

Comment:
expected to support all BoxProps here? or do we want to support styled-props only?

Reply:
Also is same change needed in .native file too?

Reply:
I think we can pass the styled props instead.

Reply:
Updated